data/scene/earth/earth.mod

```diff
@@ -63,6 +58,67 @@ return {
         },
         GuiName = "/Solar/Planets/Earth"
     },
+    -- Earth Atmosphere
+    {
+         Name = "EarthAtmosphere",
+         Parent = "Earth",
+         Renderable = {
+              Type = "RenderableAtmosphere",
+              Atmosphere = {
+                Type = "RenderablePlanet",
+                -- Atmosphere radius in Km
+                AtmosphereRadius = 6447.0,
+                --PlanetRadius    = 6378.137,
+                PlanetRadius    = 6356.752,
+                --PlanetRadius    = 6360.0,
+                PlanetAverageGroundReflectance = 0.1,
+                Rayleigh = {
+                    Coefficients = {
+                        -- Wavelengths are given in 10^-9m
+                        Wavelengths = {680, 550, 440},
+                        -- Reflection coefficients are given in km^-1
+                        Scattering = {5.8E-3, 13.5E-3, 33.1E-3},
+                        -- In Rayleigh scattering, the coefficients of absorption and scattering are the same.
+                    },
+                    -- Thichkness of atmosphere if its density were uniform, in Km
+                    H_R = 8.0,
+                },
+                --[[
+                Ozone = {
+                     Coefficients = {
+                        -- Extinction coefficients 
+                        Extinction = {3.426E-5, 8.298E-5, 0.356E-5}
+                     },
+                     H_O = 8.0,
+                },
+                ]]--
+                -- Default
+                Mie = {
+                    Coefficients = {
+                        -- Reflection coefficients are given in km^-1
+                        Scattering = {4.0e-3, 4.0e-3, 4.0e-3},
+                        --Scattering = {2.0e-5, 2.0e-5, 2.0e-5},
+                        -- Extinction coefficients are a fraction of the Mie coefficients
+                        Extinction = {4.0e-3/0.9, 4.0e-3/0.9, 4.0e-3/0.9}
+                    },
+                    -- Height scale (atmosphere thickness for constant density) in Km
+                    H_M = 1.2,
```

Comment:
Just FYI: The word "scale height" is commonly used in atmospheric sciences. If there are different scale heights for different variables, these scale heights are called "{varname} scale height", e.g., "density scale height" or "pressure scale height". For the symbol of scale height, H is commonly used.